third_party/qpg_sdk/qpg_sdk.gni

```diff
@@ -102,6 +102,7 @@ template("qpg_sdk") {
     cflags += [
       "-Wno-maybe-uninitialized",
       "-Wno-shadow",
+      "-Wno-error=array-parameter",
```

Comment:
This did not actually seem to help with QPG platform. I though I removed this in the most recent version. If this is actually required please let me know. Otherwise this will be removed in a subsequent version